app/models/drupal_content_field_image_gallery.rb

```diff
@@ -1,27 +0,0 @@
-require 'php_serialize'
-
-class DrupalContentFieldImageGallery < ApplicationRecord
```

Comment:
OK, so I think it didn't work because we deleted this entire file. So we either need to re-add it, or we need to do raw SQL. I think I will re-add this file so that the migration can succeed.

Reply:
phew, a lot of editing on main, but fingers X this works: https://github.com/publiclab/plots2/commit/cb073aedc40fcb4f1413e03366a8c6f9e8d73ad5

Reply:
https://ci-j.publiclab.org/job/Plots-Stable/3311/console